pkgs/by-name/fi/firmware-manager/package.nix

```diff
@@ -23,13 +23,8 @@ stdenv.mkDerivation rec {
     hash = "sha256-Q+LJJ4xK583fAcwuOFykt6GKT0rVJgmTt+zUX4o4Tm4=";
   };
 
-  cargoDeps = rustPlatform.importCargoLock {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "ecflash-0.1.0" = "sha256-W613wbW54R65/rs6oiPAH/qov2OVEjMMszpUJdX4TxI=";
-      "system76-firmware-1.0.51" = "sha256-+GPz7uKygGnFUptQEGYWkEdHgxBc65kLZqpwZqtwets=";
-    };
-  };
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-LooE5jU4G1QHYTa/sB95W6VJs7lY7sjHI9scUaZRmq4=";
```

Comment:
I don't think this works with `mkDerivation`, but not sure though

Reply:
you are correct. I was just passing through all my opened MRs. thanks for your thorough reviews.